http-filter-example/header_processor.cc

```diff
@@ -0,0 +1,58 @@
+#include "header_processor.h"
+
+namespace Envoy {
+namespace Http {
+
+    SetHeaderProcessor::SetHeaderProcessor() {}
+
+    int SetHeaderProcessor::parseOperation(std::vector<absl::string_view>& operation_expression) {
```

Comment:
If we're only ever returning the same number for "malformed expression," maybe a bool would be better?

I also see other extensions using `absl::Status`: https://github.com/envoyproxy/envoy/blob/main/source/extensions/filters/http/composite/factory_wrapper.cc#L12
https://abseil.io/docs/cpp/guides/status

At this scale, it's not a big deal. But at some point as this gets bigger, we'll probably want to move to `absl::Status`

Reply:
Ideally it would be good to return a few different numbers based on different types of errors -- will switch to absl::Status, thank you for pointing me to it!